arabot/modules/games.py

```diff
@@ -476,7 +477,7 @@ async def voting():
         and [m.bot for m in vc.members].count(False) > 2  # pylint: disable=used-before-assignment
     )
     @commands.cooldown(1, 120, commands.BucketType.guild)
-    @commands.command(aliases=["impostor", "impasta"])
+    @commands.command(aliases=["impostor", "impasta"], brief="Eject imposter from voice chat")
```

Comment:
Is impost**o**r/impost**e**r a deliberate typo here?

Reply:
Both are acceptable variants of the word.